### DISCUSSION TITLE
Expose `createSqlitePersister` in package for custom SQLite set-ups

### DISCUSSION THREAD

**Discussion by hdoro** [tinyplex/tinybase#166] [Ideas]
I'm trying to use Tinybase with Tauri. I can store my data as a simple custom persister that saves a JSON of the Tinybase store, but that has a bigger overhead as I'd be writing/reading the _entire_ data at every operation.

Instead, I'd like to leverage SQLite's smartness via [Tauri's SQL plugin](https://beta.tauri.app/plugin/sql/). From reading the sqlite persisters' code, I can see it leverages an internal `createSqlitePersister`, which I imagine could handle tauri/sql's `db.execute` (syntax from [sqlx](https://docs.rs/sqlx/latest/sqlx/))?

As that's an internal module, I can't access it, and it's too tied into the repo to simply copy and paste, so I haven't tried it yet - could it work? 🤔

**Comment by jamesgpearce** [comment]
OK! I refactored a bunch of things for 5.2 (Postgres support!) and iight be able to use that to your advantage.

**Reply by hdoro** [reply]
Thanks a ton, James! Excited for 5.2 :D

**Comment by jamesgpearce** [comment]
Tracked as issue #168

**Comment by jamesgpearce** [comment]
Closed out in beta 5 - wanna check it out? #168
